.pre-commit-hooks.yaml

```diff
@@ -2,6 +2,7 @@
   name: Check for PII
   description: this hook checks if modified files have PII and marks it.
   entry: pii_check
-  pass_filenames: false
```

Comment:
This is important if we want to leverage the file built-in pre-commit filtering feature (see below).